_pytest/python.py

```diff
@@ -976,8 +977,23 @@ def _idval(val, argname, idx, idfn):
                 return s
         except Exception:
             pass
+
+    try:
+        import enum
+    except ImportError:
+        # Only available in Python 3.4+
+        enum = None
```

Comment:
You can put a `#pragma: no cover` comment on this line to keep code coverage happy here. :smile: